### DISCUSSION TITLE
Request approved but did not receive benefits even after 5 days.

### DISCUSSION THREAD

**Discussion by Chandravo** [community/community#134674] [GitHub Education]
### Hi everyone,

I got my student status approved on July 30, 2024, and today is August 5 2024 and I am yet to receive the student benefits.

**Comment by prabhjotschugh** [comment]
Hi @Chandravo ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks;

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @Chandravo  was approved and your benefits are now available.`

For further reference you can look at these discussions: [#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470) , [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
